doc/user-guide/information-model.md

```diff
@@ -116,6 +119,57 @@ This section specifies what a valid catalog, workflow, and flow conditions look
 |`:lifecycle/calls`       |`keyword`                     | no       |
 |`:lifecycle/doc`         |`string`                      | yes      |
 
+### Windows
+
+- a single Clojure vector which is EDN serializable/deserializable
+- all elements in the vector must be Clojure maps
+- allows arbitrary key/values in the map as parameters
+
+| key name             |type       | optional?|
+|----------------------|-----------|----------|
+|`:window/id`          |`keyword`  | no       |
+|`:window/task`        |`keyword`  | no       |
+|`:window/type`        |`string`   | no       |
+|`:window/aggregation` |`string`   | no       |
+|`:window/window-key`  |`string`   | no       |
+|`:window/min-value`   |`int`      | yes      |
+|`:window/range`       |`vector`   | no       |
+|`:window/slide`       |`vector`   | sometimes|
+|`:window/init`        |`string`   | sometimes|
+|`:window/doc`         |`string`   | yes      |
+
+`:window/range` and `:window/slide` are values that require Units. See below for a description of Units.
```

Comment:
It'd be nice have some examples when we describe the information model, though I guess these could be left for the doc that discusses windowing/triggers.